src/main/java/edu/ucsb/cs156/organic/controllers/SchoolController.java

```diff
@@ -96,7 +95,7 @@ public School getById(
 
     @Operation(summary= "Delete a school")
     @PreAuthorize("hasRole('ROLE_ADMIN')")
-    @DeleteMapping("")
+    @DeleteMapping("/delete")
```

Comment:
```suggestion
    @DeleteMapping("")
```

This is fine. There is not need to add `/delete`